tests/generator-Tests/Unit-Tests/CodeGeneratorExpectedResults/JavaInterop1/WriteClass.txt

```diff
@@ -113,13 +113,16 @@ public partial class MyClass {
 
 	public abstract int AbstractCount {
 		// Metadata.xml XPath method reference: path="/api/package[@name='java.code']/class[@name='MyClass']/method[@name='get_AbstractCount' and count(parameter)=0]"
+		[global::Java.Interop.JniMethodSignature ("get_AbstractCount", "()I")]
```

Comment:
Do we need to update the Linker to remove these attributes?  Otherwise the size of binding assemblies may go up, resulting in larger `.apk`s.

Reply:
If these attributes were to be used in Android, *yes* we'd want to remove them.

However, unless someone is using the JavaInterop1 codegen target -- which is *highly* unlikely, considering *it doesn't really work right* (as evidenced by the fact that the Java.Base binding is excluding almost everything) -- then there won't be any attributes to remove.

This is thus a more "eventually…" thing to do, rather than something to worry about *now*.  (.NET 8 timeframe?  Maybe?  Perhaps?)

Reply:
👍 Got confused with `XAJavaInterop1`.